modules/p2/ting_openlist/include/ting_openlist_analysis.inc

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * @file
+ * Functions for analysis of local OpenList Entities
+ */
+
+function ting_openlist_get_local_analysis() {
+  $data = ting_openlist_get_local_analysis_data();
+  return '<pre>' . htmlentities(var_export($data, TRUE)) . '</pre>';
+}
```

Comment:
I do not think this function provides much value on its own. It seems directly tied to `ting_openlist_show_local_analysis_form_callback()` so the logic might as well be placed there.